src/pages/trader-tools/pnl-for-multipliers/_example-pnl-multipliers.tsx

```diff
@@ -491,9 +491,11 @@ export const TakeProfitAmountUp = () => {
                             <FormulaValue>
                                 <Localize translate_text="((3400" />
                                 <PointerContainer width="200px" ml="-75px" height="89px">
-                                    <PointerDot />
-                                    <PointerStick height="62px" />
-                                    <PointerText>
+                                    <PointerDot ml="100px" />
+                                    <PointerStick ml="100px" height="31px" />
+                                    <PointerStickHorizontal w="125px" ml="-25px" />
```

Comment:
Let's avoid using negative values for `margin` and `padding`.

Reply:
In current implementation it's impossible due the reusable component for each example.